ceph_diagnostics_collect.sh

```diff
@@ -431,12 +431,12 @@ get_orch_info() {
 
     info "collecting orchestrator info ..."
 
-    store    ${t}-status        ${CEPH} orch status
-    store    ${t}-ls            ${CEPH} orch ls
-    store -s ${t}-ls-export     ${CEPH} orch ls --export
-    store -S ${t}-ls_yaml       ${CEPH} orch ls --format yaml
-    store    ${t}-ps            ${CEPH} orch ps
-    store    ${t}-host          ${CEPH} host ls
+    store    ${t}-status ${CEPH} orch status
+    store    ${t}-ls ${CEPH} orch ls
+    store -s ${t}-ls-export ${CEPH} orch ls --export
+    store -S ${t}-ls_yaml ${CEPH} orch ls --format yaml
+    store    ${t}-ps ${CEPH} orch ps
+    store    ${t}-host ${CEPH} host ls
```

Comment:
Sorry, I was not clear. I meant to preserve space alignment (it is more readable in my opinion)  but just remove unnecessary extra spaces. Anyway, I am fine with any variant.